pkg/logs/logs.go

```diff
@@ -53,6 +94,18 @@ func (o *LogsClient) DevModeLogs(componentName string, namespace string) ([]map[
 		}
 	}
 
+	if !ignorePods {
```

Comment:
Do we need this flag? We know that the Dev mode does not create pods directly

Reply:
But the Pods created by Dev mode have the labels that match the `selector` we generate above. As a result, if we remove this flag, the logs for Pods created by Dev mode show up twice.

OTOH, in Deploy mode, Pods aren't created with these labels unless a k8s component in Devfile is `kind: Pod`.